packages/flutter_tools/lib/src/asset.dart

```diff
@@ -236,17 +247,16 @@ class ManifestAssetBundle implements AssetBundle {
     // device.
     _lastBuildTimestamp = DateTime.now();
     if (flutterManifest.isEmpty) {
-      entries[_kAssetManifestJsonFilename] = DevFSStringContent('{}');
-      entryKinds[_kAssetManifestJsonFilename] = AssetKind.regular;
+      entries[_kAssetManifestJsonFilename] = AssetBundleEntry(DevFSStringContent('{}'));
       final ByteData emptyAssetManifest =
         const StandardMessageCodec().encodeMessage(<dynamic, dynamic>{})!;
-      entries[_kAssetManifestBinFilename] =
-        DevFSByteContent(emptyAssetManifest.buffer.asUint8List(0, emptyAssetManifest.lengthInBytes));
-      entryKinds[_kAssetManifestBinFilename] = AssetKind.regular;
+      entries[_kAssetManifestBinFilename] = AssetBundleEntry(
+        DevFSByteContent(emptyAssetManifest.buffer
+            .asUint8List(0, emptyAssetManifest.lengthInBytes)),
```

Comment:
```suggestion
        DevFSByteContent(
          emptyAssetManifest.buffer.asUint8List(0, emptyAssetManifest.lengthInBytes),
        ),
```

Reply:
i think in general, where an expression with nested function invocations don't fit on a single line, add trailing commas for readability even if there is only a single argument